### PR TITLE
forceFlag was never used to recreate client with the new access token

### PR DESCRIPTION
**Pull request by brunovg**

### CODE REVIEW

File: googledrive/src/main/java/org/assimbly/googledrive/GoogleDriveConsumer.java [hunk 1]
```diff
@@ -95,7 +95,7 @@ protected int poll() throws Exception {
                 postProcess(fileModel, file);
             }
         } catch (GoogleJsonResponseException e) {
-            LOG.error("Google Drive Consume error:", e);
+            LOG.info(" >> GoogleJsonResponseException occurred");
             service = this.endpoint.getClient(true);
```
Comment (skin27) [inline googledrive/src/main/java/org/assimbly/googledrive/GoogleDriveConsumer.java:99]: Not good to set an exception in an info log. Better to change it into a warning. Also, when you get this as user, you want to have more info. I would suggest something like this (pseudocode):

if(connectionAttempts < 60){
  LOG.warn("Could not connect to Google Drive directory + directory. Try again after x seconds (attempt " + connectionAttempts + " of 60)");
  connectionAttempts += 1;
  service = this.endpoint.getClient(true);  
}else{
  LOG.error("GoogleJsonResponseException occurred. Connect to Google Drive directory + directory failed:", e);
}

Comment (skin27) [reply googledrive/src/main/java/org/assimbly/googledrive/GoogleDriveConsumer.java:99]: x = the poll interval. The user can change the poll interval to try it for a longer period.

File: googledrive/src/main/java/org/assimbly/googledrive/GoogleDriveEndpoint.java [hunk 2]
```diff
@@ -72,7 +76,8 @@ public boolean isSingleton() {
     }
 
     Drive getClient(boolean forceFlag) {
-        if (client == null) {
+        if (client == null || forceFlag) {
+            LOG.info(" >> Get client");
```
Comment (skin27) [inline googledrive/src/main/java/org/assimbly/googledrive/GoogleDriveEndpoint.java:80]: This line doesn't say that much. Either leave it away or provide more info what's going on:

 LOG.info("Get GoogleDrive client with clientid=" + );

File: googledrive/src/main/java/org/assimbly/googledrive/GoogleDriveEndpoint.java [hunk 3]
```diff
@@ -22,6 +24,8 @@
 @SuppressWarnings("PackageAccessibility")
 public class GoogleDriveEndpoint extends ProcessorEndpoint {
 
+    private static final Logger LOG = LoggerFactory.getLogger(GoogleDriveEndpoint.class);
+
```
Comment (skin27) [inline googledrive/src/main/java/org/assimbly/googledrive/GoogleDriveEndpoint.java:28]: There's a more generic (and I think recommended way) to get the logger:

	protected Logger log = LoggerFactory.getLogger(getClass());

**Review by skin27 — CHANGES_REQUESTED** [review]

File: googledrive/src/main/java/org/assimbly/googledrive/GoogleDriveConsumer.java (hunk 1 above)
Comment (brunovg) [reply googledrive/src/main/java/org/assimbly/googledrive/GoogleDriveConsumer.java]: changes done!
x = should be the delay parameter set by the user

File: googledrive/src/main/java/org/assimbly/googledrive/GoogleDriveEndpoint.java (hunk 2 above)
Comment (brunovg) [reply googledrive/src/main/java/org/assimbly/googledrive/GoogleDriveEndpoint.java]: removed this log

File: googledrive/src/main/java/org/assimbly/googledrive/GoogleDriveEndpoint.java (hunk 3 above)
Comment (brunovg) [reply googledrive/src/main/java/org/assimbly/googledrive/GoogleDriveEndpoint.java]: deleted line. don't need anymore

**Review by skin27 — APPROVED** [review]

**Pull request merged**